### DISCUSSION TITLE
What's version of  "@angular-eslint/schematics" is compatible with angular 16?

### DISCUSSION THREAD

**Discussion by amalamalamal** [angular-eslint/angular-eslint#1693] [General]
when i was installing angular/cli 16 I had this issue: package "@angular-eslint/schematics" has an incompatible peer dependency to "@angular/cli"(requires ">14.0.0 < 15.0.0", would install "16.2.11")

**Comment by chimurai** [comment]
For Angular 16 you'll need `@angular-eslint/schematics@16`

Version compatibility is documented here:
https://github.com/angular-eslint/angular-eslint#supported-angular-cli-versions
